### PR TITLE
Optimized bitset for all bits set

### PR DESCRIPTION
**Pull request by Aceeri**
Just a specialized bit set for everything being set.

### CODE REVIEW

**Review by Xaeroxe — APPROVED** [review]
❤️ I love it!  Thank you!

File: src/ops.rs [hunk 1]
```diff
@@ -238,6 +238,33 @@ impl<A: BitSetLike, B: BitSetLike> BitSetLike for BitSetXor<A, B> {
...
 }
 
+/// `BitSetAll` is a bitset with all bits set. Essentially the same as
+/// `BitSetNot(BitSet::new())` but without any allocation.
+#[derive(Debug)]
+pub struct BitSetAll;
+impl BitSetLike for BitSetAll {
+    #[inline]
+    fn layer3(&self) -> usize {
+        ::std::usize::MAX
```
Comment (torkleyy) [inline src/ops.rs]: meh, please use `!0` or import

**Review by torkleyy — APPROVED** [review]
Otherwise :+1:

**Comment by kvark** [comment]
bors r=torkleyy

**Pull request merged**